src/Exception/InvalidMutatorException.php

```diff
@@ -48,7 +48,7 @@ public static function create(string $filePath, Mutator $mutator, \Throwable $pr
             sprintf(
                 'Encountered an error with the "%s" mutator in the "%s" file. ' .
                 'This is most likely a bug in Infection, so please report this in our issue tracker.',
-                $mutator::getName(),
+                $mutator->getName(),
```

Comment:
What's the goal of this change?

Reply:
With that method being static, it becomes a lot harder (impossible?) to make a proper decorator for mutators.